qtt/instrument_drivers/virtualAwg/sequencer.py

```diff
@@ -58,6 +58,32 @@ def make_square_wave(amplitude, period, repetitions=1, name='pulse'):
         return {'name': name, 'wave': SequencePT(*(sequence_data,)*repetitions),
                 'type': DataTypes.QU_PULSE}
 
+    @staticmethod
+    def make_pulse_table(amplitudes, waiting_times, repetitions=1, name='pulse_table'):
+        """ Creates a sequence of pulses from a list of amplitudes and waiting times.
+            Note that the initial voltage level will be given by the last element in amplitudes.
+
+        Args:
+            amplitudes (list of floats): List with voltage amplitudes of the pulses.
+            waiting_times (list of float): List with durations containing the waiting time of each pulse.
+            repetitions (int): The number of oscillations in the sequence.
```

Comment:
```suggestion
            repetitions (int): The number of repetitions of the base sequence.
```
Oscillations is not the correct term